src/analysis/Environments/Environments.test.ts

```diff
@@ -271,7 +266,8 @@ describe('Environments', () => {
       const buttons2 = getAllByRole(runtime2ButtonsCell, 'button');
       expect(buttons2.length).toBe(2);
       expect(buttons2[0].textContent).toBe('Pause');
-      expect(buttons2[0].getAttribute('aria-disabled')).toBe('true');
+      // TODO: Back to true once https://broadworkbench.atlassian.net/browse/PROD-905 is resolved
```

Comment:
Does it make sense to add this to the ticket to not lost the reminder that in order to close the ticket we have to switch this value?

Reply:
I can leave a message in the ticket to clean up the TODOs yep :) This might end up being a more permanent solution, we don't know yet